README.md

```diff
@@ -1,229 +1,151 @@
 
-### [Code Style](./.github/workflows/style.yml)
+### Check R Tags
 
-[![Style](https://github.com/pharmaverse/admiralci/actions/workflows/style.yml/badge.svg)](https://github.com/pharmaverse/admiralci/actions/workflows/style.yml)
+* [Link to workflow code (yml
+  file)](https://github.com/pharmaverse/admiralci/blob/main/.github/workflows/check-r-tags.yml)
+* [Link to the
+  documentation](https://pharmaverse.github.io/admiralci/articles/check-r-tags.html)
 
-Code style is enforced via the [`styler`](https://styler.r-lib.org/) R
-package. Custom style configurations, if any, will be honored by this
-workflow.
+# How to use these workflows?
 
-Failed workflows are indicative of unstyled code.
+## Reuse (recommended)
 
-### [CRAN Status](./.github/workflows/cran-status.yml)
-
-[![CRAN
-Status](https://github.com/pharmaverse/admiralci/actions/workflows/cran-status.yml/badge.svg)](https://github.com/pharmaverse/admiralci/actions/workflows/cran-status.yml)
-
-This workflow allows you to monitor the current status of checks on
-CRAN, if your package has been published on CRAN. Given that CRAN has
-its own systems on which it runs checks and it might not be possible to
-emulate all of their checks using GitHub Actions or CI/CD, it’s just
-more efficient to let their systems run tests and be notified via a
-GitHub issue if an error is encountered.
-
-Usually CRAN sends out email notifications if errors are encountered
-during their checks but only individual package owners are notified and
-a stringent deadline of 2 weeks is given to remediate the errors on
-CRAN. To mitigate the risk of the package being kicked off CRAN, this
-workflow provides transparency and visibility to all project
-collaborators by creating a GitHub issue so that the errors can be
-remediated before the deadline has passed.
-
-
-### [`docker image build`](./.github/workflows/push-docker-image.yml)
-
-This workflow is pushing a docker image inside admiralci ghcr (github container registry). It's triggered each time a change is detected inside `renv.lock` files.
-A docker image is produced for each R versions (defined under `renv/profiles` folder). This image will contain every system dependencies required + `renv.lock` files packages.
-This workflow mainly uses the action `insightsengineering/r-image-creator`, which is a generic action to create `rocker/rstudio` based docker images.
-
-## How to use these workflows?
-
-### Reuse (recommended)
-
-You could add just *one* file called `.github/workflows/common.yml` to
+You could add just _one_ file called `.github/workflows/common.yml` to
 directly import these workflows while receiving the latest updates and
 enhancements, given that the workflows defined in this repository are
-reusable via the
-[`workflow_call`](https://docs.github.com/en/actions/using-workflows/reusing-workflows)
-GitHub Actions event.
+reusable via the \[`workflow_call`]\[workflow\_call] GitHub Actions
```

Comment:
@dgrassellyb is it that I don't understand what this syntax is saying or is it a typo? Don't be afraid to tell me if it's the former 😄

Reply:
Indeed since to comes from .Rmd -> .md job, I have this in the .Rmd : 

 reusable via the [`workflow_call`][workflow_call] GitHub Actions event.

Reply:
ok, thanks!